src/pheval_exomiser/utils/phenopacket_utils.py

```diff
@@ -0,0 +1,138 @@
+import json
+import os
+from collections import defaultdict
+from dataclasses import dataclass
+from pathlib import Path
+
+from google.protobuf.json_format import Parse
+from phenopackets import Family, Phenopacket
+
+from ..prepare.custom_exceptions import IncorrectFileFormatError
+
+
+class IncompatibleGenomeAssemblyError(Exception):
+    """Exception raised for incompatible genome assembly."""
+
+    def __init__(self, assembly, phenopacket, message="Incompatible Genome Assembly"):
+        self.assembly: str = assembly
+        self.phenopacket: str = phenopacket
+        self.message: str = message
+        super().__init__(self.message)
+
+    def __str__(self):
+        return f"{self.message} -> {self.assembly} in {self.phenopacket}"
+
+
+@dataclass
+class CausativeVariant:
+    phenopacket: Path
+    proband_id: str
+    assembly: str
+    chrom: str
+    pos: str
+    ref: str
+    alt: str
+    genotype: str
+
+
+class PhenopacketReader:
```

Comment:
Shouldnt this be in the pheval utils main package?

Reply:
Of course, this is what I am planning on merging into the main pheval utils repo. This code is just here to make sure the functions are still working as expected. Once I have the main pheval methods merged in, this structure will change again and duplicated files or methods will be removed and instead imported from the main package. This is still very much a work in progress, I am very much focusing on creating the tests for the main pheval functions so that it can be merged in and I can fix this repo to where it needs to be after that

Reply:
Yes. This package is duplicated - we need [the other PR](https://github.com/monarch-initiative/pheval/pull/30) merging so this can be removed.

Reply:
Ok great thank you! I will let you work it out and we can do a general review once everything is working?